drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -60,9 +60,22 @@
 
 #import <Metal/MTLTexture.h>
 #import <Metal/Metal.h>
+#import <os/log.h>
+#import <os/signpost.h>
 #import <spirv_msl.hpp>
 #import <spirv_parser.hpp>
 
+#pragma mark - Logging
+
+os_log_t LOG_DRIVER;
+// Used for dynamic tracing.
+os_log_t LOG_INTERVALS;
+
+__attribute__((constructor)) static void InitializeLogging(void) {
+	LOG_DRIVER = os_log_create("org.stuartcarnie.godot.metal", OS_LOG_CATEGORY_POINTS_OF_INTEREST);
```

Comment:
Was this meant to be hardcoded?

Reply:
No, I'm switching that to `org.godotengine.godot.metal` in a follow-up PR

Reply:
Heh good catch, I was eager to merge this for 4.4.dev1 so I overlooked it.
Stuart gets one dev snapshot of fame in the log uid ;)

Reply:
I've correct that in #96089 too 😊

Reply:
Thanks for the spot, @zeux!